btk/measure.py

```diff
@@ -89,7 +89,7 @@ def sep_measure(batch, idx):
 
     image = batch["blend_images"][idx]
     stamp_size = image.shape[-2]  # true for both 'NCHW' or 'NHWC' formats.
-    coadd = np.mean(image, axis=0)
+    coadd = np.mean(image, axis=np.argmin(image.shape))  # Smallest dimension is the channels
```

Comment:
Yes, I tried to do it without having any additional information in input. Maybe adding the keyword would be better.

Reply:
good point, I hand't realized this depended on "NCHW" or "NHWC" convention. Initially we wanted all `measure_functions` to only take in two arguments `batch` and `idx`, now I see that this might be too restrictive for some cases. 

One idea is to allow for `measure_functions` to have keywords and pass them in the initialization of the `measure_generator`. Another alternative is to use `partial`

Reply:
This can also be done in a separate PR addressing #133

Reply:
I pushed a change to address this in PR #136 , feel free to review if you'd like @thuiop